projects/storefrontapp-e2e-cypress/cypress/e2e/regression/my-account/update-password.e2e-flaky.cy.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as asm from '../../../helpers/asm';
+import * as checkout from '../../../helpers/checkout-flow';
+import * as alerts from '../../../helpers/global-message';
+import { signOutUser } from '../../../helpers/login';
+import * as updatePassword from '../../../helpers/update-password';
+import { generateMail, randomString } from '../../../helpers/user';
+import { viewportContext } from '../../../helpers/viewport-context';
+import { standardUser } from '../../../sample-data/shared-users';
+import { isolateTests } from '../../../support/utils/test-isolation';
+describe('My Account - Update Password', () => {
+  viewportContext(['mobile'], () => {
+    before(() =>
+      cy.window().then((win) => {
+        win.sessionStorage.clear();
+      })
+    );
+    // Core e2e test. Repeat in mobile viewport.
+    updatePassword.testUpdatePasswordLoggedInUser();
+  });
+
+  viewportContext(['mobile', 'desktop'], () => {
+    before(() =>
+      cy.window().then((win) => {
+        win.sessionStorage.clear();
+      })
+    );
+
+    describe('update password by agent', { testIsolation: false }, () => {
+      isolateTests();
+      before(() => {
+        standardUser.registrationData.email = generateMail(
+          randomString(),
+          true
+        );
+        cy.requireLoggedIn(standardUser);
+        cy.visit('/');
+      });
+
+      it('should display server error if agent try to modify customer password', () => {
```

Comment:
please add traceability to the test, append (CXSPA-XXXX) to the end of the description

Reply:
aggree